website/docs/r/cis_firewall.html.markdown

```diff
@@ -31,57 +31,73 @@ resource "ibm_cis_firewall" "lockdown" {
   }
 }
 ```
-**NOTE:** IBM terraform provider supports only lockdowns rules
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `cis_id` - (Required,string) The ID of the CIS service instance
-* `domain_id` - (Required,string) The ID of the domain to add the Lockdown.
-* `firewall_type` - (Required,string) The type of firewall. Allowable values are [`lockdowns`],[`access_rules`],[`ua_rules`].
+- `cis_id` - (Required,string) The ID of the CIS service instance
+- `domain_id` - (Required,string) The ID of the domain to add the Lockdown.
+- `firewall_type` - (Required,string) The type of firewall. Allowable values are [`lockdowns`],[`access_rules`],[`ua_rules`].
+
+**NOTE:**
 
-**NOTE:** 
 1. [`access_rules`]: Access Rules are a way to allow, challenge, or block requests to your website. You can apply access rules to one domain only or all domains in the same service instance.
 2. [`ua_rules`]: Perform access control when matching the exact UserAgent reported by the client. The access control mechanisms can be defined within a rule to help manage traffic from particular clients. This will enable you to customize the access to your site.
 3. [`lockdowns`]: Lock access to URLs in this domain to only permitted addresses or address ranges.
 
-* `lockdown` - (Optional,list) (Maxitems: 1) List of lockdown to be created. It is the data describing a lockdowns rule.
-    * `paused` - (Optional,bool).  Whether this rule is currently disabled.
-    * `description` - (Optional,string). Some useful information about this rule to help identify the purpose of it.
-    * `priority` - (Optional,int) The priority of the record
-    * `urls` - (Optional,list). URLs included in this rule definition. Wildcards are permitted. The URL pattern entered here is escaped before use. This limits the URL to just simple wildcard patterns.
-    * `configurations` - (Optional,list). List of IP addresses or CIDR ranges to use for this rule. This can include any number of [`ip`] or [`ip_range`] configurations that can access the provided URLs.
-        * `target` - (Optional,string). The request property to target. Valid values: [`ip`], [`ip_range`].
-        * `value` - (Optional,string). IP addresses or CIDR. If target is [`ip`], then value is an IP addresses, otherwise CIDR. 
-
-**NOTE:**  `lockdowns` argument is a required parameter if `firewall_type` is [`lockdowns`]. To create a firewall-lockdown `paused`,`urls`,`configurations` are required arguments. For `firewall_type` other than [`lockdowns`], `lockdowns` argument shouldn't be passed in a payload
+- `lockdown` - (Optional,list) (MinItems: 1) List of lockdown to be created. It is the data describing a lockdowns rule.
```

Comment:
Please mention exactly one of should be given and these variable should be given according to firewall type

Reply:
changed the comments